src/OpenLoco/Date.h

```diff
@@ -24,13 +26,13 @@ namespace OpenLoco
     {
         int32_t day = 0;
         MonthId month;
-        int32_t year = 0;
+        Year year = 0;
```

Comment:
Unsure if this is safe to change. Will need to investigate all users of the structure to ensure its not writing/reading any size fixed areas.

Reply:
no worries, this was the most speculative of my PRs for v22.02, happy to park this PR if we want because it doesn't really 'fix' anything that I know of, it was just to improve type consistency